src/Mvc/Mvc.Core/ref/Microsoft.AspNetCore.Mvc.Core.netcoreapp.cs

```diff
@@ -3256,7 +3256,7 @@ public partial class ValidationVisitor
         {
             private readonly object _dummy;
             private readonly int _dummyPrimitive;
-            public StateManager(Microsoft.AspNetCore.Mvc.ModelBinding.Validation.ValidationVisitor visitor, object newModel) { throw null; }
+            public StateManager(Microsoft.AspNetCore.Mvc.ModelBinding.Validation.ValidationVisitor visitor) { throw null; }
```

Comment:
This is a breaking change to a public API (protected nested class of a public class). Can you add back the old constructor even if it's not used? We try to avoid API breaks when they are low value.

Reply:
sure, makes sense!